packages/core/source/polyfill.ts

```diff
@@ -72,4 +72,4 @@ hooks.removeAttribute = (element, name) => {
   updateRemoteElementAttribute(element, name);
 };
 
-export {hooks, window, type Hooks};
+export {hooks, window, Window, type Hooks};
```

Comment:
I exported `Window` for convenience, so the patch for Admin can be a bit cleaner:

```js
import {window, Window} from '@remote-dom/core/polyfill';

Window.setGlobalThis(window);
```

vs

```js
import {window} from '@remote-dom/core/polyfill';
import {Window} from '@remote-dom/polyfill';

Window.setGlobalThis(window);
```